packages/govuk-frontend-review/src/views/layouts/_generic.njk

```diff
@@ -15,6 +19,9 @@
 {% set mainClasses = 'govuk-main-wrapper--auto-spacing' %}
 
 {% block bodyEnd %}
-  <script type="module" src="/javascripts/all.min.js"></script>
-  <script type="module">window.GOVUKFrontend.initAll()</script>
+  <script type="module" src="/javascripts/all.bundle.min.mjs"></script>
```

Comment:
Keeping the `<script>` tag allows [browser preload scanners](https://web.dev/preload-scanner/) to download our ES module bundle early

It's optional and can be replaced with [`modulepreload`](https://developer.chrome.com/blog/modulepreload/) in future (Safari 17+)

```html
<link rel="modulepreload" href="/javascripts/all.bundle.min.mjs">
```